en/docs/developer-guides/tools/migration-tools/logic-apps-migration-tool.md

```diff
@@ -0,0 +1,463 @@
+# Azure Logic Apps migration tool
```

Comment:
Add the parameters and other important info in https://central.ballerina.io/wso2/tool_migrate_logicapps/1.0.0#parameters to this page.  Readers are not supposed to use central information. Refer to the Mule and tibco pages.

Reply:
Addressed review suggestions